grantnav/frontend/templates/search.html

```diff
@@ -88,7 +88,7 @@ <h4 class="smaller-top-margin-heading"> Current Filters </h4>
     {% endif %}
 
     <div class="col-xs-3 filters">
-      <h3 class="indent"> Filter By </h3>
+      <h3 class="indent"> Filter By <a class="filter_by_help_link" href="/help#filters"><sup><img src="{% static 'images/icon-help-50.png'%}" width="15" height="15"></sup></a></h3>
```

Comment:
What does this `class=` do?

Reply:
@robredpath This class was already in the code. It is in the CSS/main.css
.indent {
  margin-left: 15px
}

Reply:
Sorry, I mean `filter_by_help_link`. Surely it should be an ID?